### PR TITLE
US118553 - Add COA Manual Override Page to consistent-eval

### PR DESCRIPTION
**Pull request by max-hao-d2l**
The Manual Override Page for Gradebook Mastery View will use the `consistent-evaluation` component. It uses components from `d2l-outcomes-overall-achievement` in the left panel and `d2l-outcomes-level-of-achievement-ui` on the right panel.

Current Functionality:
* Displays the user's outcome progress on the left panel
* Displays an outcomes selector on the right panel to evaluate/override the overall achievement

Next Steps:
* Verify functionality alongside the Outcomes Gradebook Table component and the Gradebook API
* [US117787](https://rally1.rallydev.com/#/detail/userstory/398566207612?fdp=true): Confirm Leaving without Saving Changes
* [US117788](https://rally1.rallydev.com/#/detail/userstory/398566216352?fdp=true): Page Footer Save/Publish buttons on Manual Eval Page
* [US118921](https://rally1.rallydev.com/#/detail/userstory/408254862364?fdp=true): Navigation Bar in Manual Eval Page

https://rally1.rallydev.com/#/detail/userstory/404538971824?fdp=true

### CODE REVIEW

**Review by rgoessl — APPROVED** [review]

File: demo/data/outcomes-overall-achievement/evaluation.json [hunk 1]
```diff
@@ -42,6 +42,30 @@
             ]
         }
     ],
+    "actions": [
+        {
+            "href": "https://42a19097-12f4-4fa9-9739-146012e13bce.activities.api.proddev.d2l/activities/6606_2000_3/usages/6609/users/user_169/evaluation?action=update",
```
Comment (rgoessl) [inline demo/data/outcomes-overall-achievement/evaluation.json:47]: These URLs aren't actually invoked right? You should remove them

**Review by rgoessl — APPROVED** [review]

File: components/left-panel/consistent-evaluation-left-panel.js [hunk 2]
```diff
@@ -138,6 +143,15 @@ export class ConsistentEvaluationLeftPanel extends LocalizeMixin(LitElement) {
 		</div>`;
 	}
 
+	_renderOverallAchievement() {
+		return html`
+			<d2l-consistent-evaluation-outcomes-overall-achievement
+				href=${this.userProgressOutcomeHref}
+				token=${this.token}
```
Comment (jeroach) [inline components/left-panel/consistent-evaluation-left-panel.js:150]: Should token be passed as a property? e.g. `.token=${this.token}`

File: components/left-panel/consistent-evaluation-outcomes-overall-achievement.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+			href: { type: String },
+			token: { type: String }
+		};
+	}
+
+	render() {
+		return html`
+            <d2l-coa-primary-panel
+                href=${this.href}
+                token=${this.token}
```
Comment (jeroach) [inline components/left-panel/consistent-evaluation-outcomes-overall-achievement.js:17]: Same comment re: token as property

File: components/right-panel/consistent-evaluation-coa-eval-override.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+			}
+		};
+	}
+
+	render() {
+		return html`
+			<d2l-consistent-evaluation-right-panel-block no-title>
+				<d2l-outcomes-coa-eval-override
+					href=${this.href}
+					token=${this.token}>
```
Comment (jeroach) [inline components/right-panel/consistent-evaluation-coa-eval-override.js:22]: Same comment re: token as property

File: components/right-panel/consistent-evaluation-right-panel.js [hunk 5]
```diff
@@ -124,6 +134,17 @@ export class ConsistentEvaluationRightPanel extends LocalizeMixin(LitElement) {
 		return html``;
 	}
 
+	_renderCoaOverride() {
+		if (!this.hideCoaOverride) {
+			return html`
+				<d2l-consistent-evaluation-coa-eval-override
+					href=${this.coaOverrideHref}
+					token=${this.token}
```
Comment (jeroach) [inline components/right-panel/consistent-evaluation-right-panel.js:142]: Same comment re: token as property

**Review by jeroach — APPROVED** [review]

**Pull request merged**